src/main/java/edu/cornell/kfs/module/ar/document/validation/CuCustomerRule.java

```diff
@@ -0,0 +1,12 @@
+package edu.cornell.kfs.module.ar.document.validation;
+
+import org.kuali.kfs.module.ar.document.validation.impl.CustomerRule;
+
+public class CuCustomerRule extends CustomerRule {
+    
+    @Override
+    public boolean checkNameIsValidLength(String customerName) {
+        return true;
```

Comment:
Should we check the max length against the data dictionary for the name field?

Reply:
Max length is already checked with the DD checks and is not needed in this method.